geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh

```diff
@@ -19,7 +19,7 @@ GMUSER=hadoop
 # todo bootstrap from the current running location and ask the user if they want to 
 # install to /opt/geomesa ? Maybe propmpt with a default of /opt/geomesa similar to
 # how the maven release plugin works?
-GMDIR="/opt/geomesa-hbase_2.11-%%project.version%%"
+GMDIR="/opt/geomesa-hbase_2.12-%%project.version%%"
```

Comment:
Or we add a scala version property similar like the `%%project.version%%` while it seems that still don't apply well for hbase1 vs hbase2 so I just leave it "hard code“ as it is.

Reply:
Ah, yes I think this is coming from the maven pom, so we already have a `scala.binary.version` [flag](https://github.com/locationtech/geomesa/blob/main/pom.xml#L56).
```suggestion
GMDIR="/opt/geomesa-hbase_%%scala.binary.version%%-%%project.version%%"
```